server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -141,6 +145,9 @@ protected static boolean parseObjectOrDocumentTypeProperties(String fieldName, O
             if (fieldName.equals("position")) {
                 builder.position(nodeIntegerValue(fieldNode));
                 return true;
+            } else if (fieldName.equals("oid")) {
```

Comment:
Do we still need this here?

Reply:
ObjectMapper doesn't extend FieldMapper, cannot do smth like  
super.doXContentBody(builder, includeDefaults, params); and handle only specific fields